lua/nvim-treesitter/parsers.lua

```diff
@@ -1751,6 +1751,15 @@ list.strace = {
   maintainers = { "@amaanq" },
 }
 
+list.styled = {
+  install_info = {
+    url = "https://github.com/mskelton/tree-sitter-styled",
+    files = { "src/parser.c", "src/scanner.c" },
+    generate_requires_npm = true,
```

Comment:
Can't the generation be done from parser repo's side?

Reply:
That's only _if_ you generate the parser from the grammar, which you don't _need_ to do.

Reply:
The docs say this:

```
generate_requires_npm = false, -- if stand-alone parser without npm dependencies
```

Does that mean if the grammar has npm dependencies then `generate_requires_npm` should be `true`?  That's how that reads to me, and this grammar does have dependencies since it extends the CSS grammar.

I'm not an expert on this though, so very possible I'm wrong about this!

Reply:
Yes, that is correct. This field is used (and required) if and only if you do `:TSInstallFromGrammar` (which some but not all parsers require).

Reply:
Gotcha, updated the PR to remove `generate_requires_npm`